src/main/java/duke/command/AddTodoCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.util.Parser;
+import duke.util.TaskList;
+import duke.task.Todo;
+
+/**
+ * Represents the Command of adding a new todo to a task list.
+ */
+public class AddTodoCommand extends Command {
```

Comment:
Should class names be nouns? Maybe can rename to something like `TodoCommand` here?